tests/run_tests.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
Forgot about `tests/run_tests.py` womp

Reply:
Added that back as /usr/bin/env python2

Who needs to run tests anyways?